rfhub2/cli/keywords/keywords_extractor.py

```diff
@@ -45,12 +48,30 @@ def get_libraries_paths(self) -> Set[Path]:
         """
         libraries_paths = set()
         for path in self.paths:
-            libraries_paths.update(self._traverse_paths(Path(path)))
+            path = self.get_library_path_from_name(path)
+            if path:

```

Comment:
@MarcinMaciaszek  You are right that this is ugly. The other option would be to move the verification of path to _traverse_paths, but 
[that one is truly ugly as hell.](https://github.com/pbylicki/rfhub2/blob/master/rfhub2/cli/keywords/keywords_extractor.py#L54)
Other options that I can see is to use try/except or continue, but would they be better?